phpBB/styles/prosilver/template/ucp_header.html

```diff
@@ -16,86 +16,21 @@ <h2 class="ucp-title">{L_UCP}</h2>
 
 <div class="panel bg3">
 	<div class="inner">
-
-	<div style="width: 100%;">
-
-	<div id="cp-menu" class="cp-menu">
-		<div id="navigation" class="navigation" role="navigation">
-
-		<!-- IF S_PRIVMSGS -->
-			<!-- BEGIN t_block2 -->
-				<!-- IF S_PRIVMSGS and not t_block2.S_LAST_ROW -->
-				<ul>
-					<!-- IF t_block2.S_SELECTED -->
-						<li id="active-subsection" class="active-subsection"><a href="{t_block2.U_TITLE}"><span>{t_block2.L_TITLE}</span></a></li>
-					<!-- ELSE -->
-						<li><a href="{t_block2.U_TITLE}"><span>{t_block2.L_TITLE}</span></a></li>
-					<!-- ENDIF -->
-				</ul>
-				<!-- ENDIF -->
-			<!-- END t_block2 -->
-
-			<hr />
-			<!-- BEGIN folder -->
-				<!-- IF folder.S_FIRST_ROW --><ul><!-- ENDIF -->
-				<!-- IF folder.S_CUR_FOLDER -->
-					<li id="active-subsection" class="active-subsection"><a href="{folder.U_FOLDER}"><!-- IF folder.UNREAD_MESSAGES > 0 --><strong>{folder.FOLDER_NAME} ({folder.UNREAD_MESSAGES})</strong><!-- ELSE -->{folder.FOLDER_NAME}<!-- ENDIF --></a></li>
-				<!-- ELSE -->
-					<li><a href="{folder.U_FOLDER}"><span><!-- IF folder.UNREAD_MESSAGES > 0 --><strong>{folder.FOLDER_NAME} ({folder.UNREAD_MESSAGES})</strong><!-- ELSE -->{folder.FOLDER_NAME}<!-- ENDIF --></span></a></li>
-				<!-- ENDIF -->
-				<!-- IF folder.S_LAST_ROW --></ul><!-- ENDIF -->
-			<!-- END folder -->
-			<hr />
-		<!-- ENDIF -->
-
-			<ul>
-		<!-- BEGIN t_block2 -->
-			<!-- IF (S_PRIVMSGS and t_block2.S_LAST_ROW) or not S_PRIVMSGS -->
-				<!-- IF t_block2.S_SELECTED -->
-					<li id="active-subsection" class="active-subsection"><a href="{t_block2.U_TITLE}"><span>{t_block2.L_TITLE}</span></a></li>
-				<!-- ELSE -->
-					<li><a href="{t_block2.U_TITLE}"><span>{t_block2.L_TITLE}</span></a></li>
-				<!-- ENDIF -->
+		<div style="width: 100%;">
```

Comment:
Let go ahead and factor this ugly shit out why we are at it ;) maybe call it cp-wrap